From 7a0330923a3e8300f273a2031cd191d296e56d77 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Mon, 6 Aug 2018 17:26:33 +0800
Subject: [PATCH 4307/5242] MLK-19130-1 ARM64: dts: remove duplicated
 interrupt-parent

commit  e220be916423ccd5e7801257635055550f53e1ec from
https://source.codeaurora.org/external/imx/linux-imx.git

interrupt-parent are assigned to node gpc in root node, so
no need to set it again in child node which needs gpc as
the interrupt-parent.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
(cherry picked from commit f1e4158c785c8b6b74a8206d07f648495b755acf)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index eec49a6..c6ffd89 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -479,7 +479,6 @@
 			     "fsl,imx6q-uart", "fsl,imx21-uart";
 		reg = <0x0 0x30860000 0x0 0x10000>;
 		interrupts = <GIC_SPI 26 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-parent = <&gpc>;
 		clocks = <&clk IMX8MM_CLK_UART1_ROOT>,
 			<&clk IMX8MM_CLK_UART1_ROOT>;
 		clock-names = "ipg", "per";
@@ -493,7 +492,6 @@
 			     "fsl,imx6q-uart", "fsl,imx21-uart";
 		reg = <0x0 0x30880000 0x0 0x10000>;
 		interrupts = <GIC_SPI 28 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-parent = <&gpc>;
 		clocks = <&clk IMX8MM_CLK_UART3_ROOT>,
 			<&clk IMX8MM_CLK_UART3_ROOT>;
 		clock-names = "ipg", "per";
@@ -507,7 +505,6 @@
 			     "fsl,imx6q-uart", "fsl,imx21-uart";
 		reg = <0x0 0x30890000 0x0 0x10000>;
 		interrupts = <GIC_SPI 27 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-parent = <&gpc>;
 		clocks = <&clk IMX8MM_CLK_UART2_ROOT>,
 			<&clk IMX8MM_CLK_UART2_ROOT>;
 		clock-names = "ipg", "per";
-- 
1.7.9.5

